### PR TITLE
update all remaining references to yml files in docs

### PR DESCRIPTION
**Pull request by nemreid**
**Summary:** Summary of changes

Addresses [CUMULUS-1560: Develop amazing docs](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1560)

## Changes

* update all remaining references to yml files in docs

### CODE REVIEW

File: docs/troubleshooting/README.md [hunk 1]
```diff
@@ -46,32 +46,35 @@ More information on kinesis error handling is [here](data-cookbooks/cnm-workflow
 
 `KMS Exception: AccessDeniedExceptionKMS Message: The ciphertext refers to a customer master key that does not exist, does not exist in this region, or you are not allowed to access.`
 
-The above error was being thrown by cumulus lambda function invocation. The KMS key is the encryption key used to encrypt lambda environment variables. The root cause of this error is unknown.
+The above error was being thrown by cumulus lambda function invocation. The KMS key is the encryption key used to encrypt lambda environment variables. The root cause of this error is unknown, but is speculated to caused by deleting and recreating, with the same name, the IAM role the lambda uses.
 
 On a lambda level, this error can be resolved by updating the KMS Key to `aws/lambda`. We've done this through the management console. Unfortunately, this approach doesn't scale well.
```
Comment (markdboyd) [inline docs/troubleshooting/README.md]: FWIW, I investigated trying to use this solution yesterday and it doesn't seem to work or make any sense? I would vote for removing it

File: docs/deployment/thin-egress-app.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ hide_title: true
 
 The [Thin Egress App (TEA)](https://github.com/asfadmin/thin-egress-app) is an app running in Lambda that allows retrieving data from S3 using temporary links and provides URS integration.
 
+Note: If you are using the `cumulus` module, you will not need the information on this page, as Cumulus packages TEA within its `distribution` tf-module, which is pre-configured and included in our releases as a submodule of the `cumulus` module. However, if you are using individual Cumulus modules in your own Terraform configuration, or wish to configure TEA on your own, the information below is important to set up distribution.
```
Comment (Jkovarik) [inline docs/deployment/thin-egress-app.md]: 👍

**Comment by Jkovarik** [comment]
Approved, pending Mark's comment

File: docs/troubleshooting/README.md (hunk 1 above)
Comment (nemreid) [reply docs/troubleshooting/README.md]: Let me update it to what worked for Abdelhak yesterday.

Comment (nemreid) [reply docs/troubleshooting/README.md]: eb775a478

File: example/deployments/lf-test-tf.tfvars [hunk 3]
```diff
@@ -1,4 +1,4 @@
-prefix            = "lf-int-test-tf"
+prefix            = "lf-test-tf"
```
Comment (laurenfrederick) [inline example/deployments/lf-test-tf.tfvars]: You are the best!!!

File: docs/deployment/thin-egress-app.md (hunk 2 above)
Comment (laurenfrederick) [reply docs/deployment/thin-egress-app.md]: yes, nice work

**Pull request merged**